pipeline/inputs/splunk.md

```diff
@@ -12,7 +12,9 @@ The **splunk** input plugin handles [Splunk HTTP HEC](https://docs.splunk.com/Do
 | buffer_max_size          | Specify the maximum buffer size in KB to receive a JSON message.                                                                              | 4M      |
 | buffer_chunk_size        | This sets the chunk size for incoming incoming JSON messages. These chunks are then stored/managed in the space available by buffer_max_size. | 512K    |
 | successful_response_code | It allows to set successful response code. `200`, `201` and `204` are supported.                                                              | 201     |
-| splunk\_token           | Add an Splunk token for HTTP HEC.`                                  |         |
+| splunk\_token           | Add an Splunk token for HTTP HEC. If specified multiple tokens with commas, it will be divided into each of tokens and they will be used for authentication of HTTP HEC. |         |
```

Comment:
Is the division across tokens equal or is it just use the first one until it is maxed, then the second, and so on?
```suggestion
| splunk\_token           | Specify a Splunk token for HTTP HEC authentication. If multiple tokens are specified (with commas and no spaces), usage will be divided across each of the tokens. |         |
```

Reply:
With the current implementation, the division is across tokens equal.
Splunk token is able to specify only one at sender(HEC client). So, the specified tokens are traversed until the max or matched.